examples/src/main/scala/org/apache/spark/examples/ml/BinarizerExample.scala

```diff
@@ -31,7 +31,7 @@ object BinarizerExample {
       .getOrCreate()
     // $example on$
     val data = Array((0, 0.1), (1, 0.8), (2, 0.2))
-    val dataFrame: DataFrame = spark.createDataFrame(data).toDF("label", "feature")
+    val dataFrame = spark.createDataFrame(data).toDF("label", "feature")
```

Comment:
Since you're removing the type annotation, you can remove the import of `DataFrame` also.